{
  "repository": {
    "name": "fabriciofx/cactoos-jdbc",
    "githubId": 132658324,
    "url": "https://github.com/fabriciofx/cactoos-jdbc",
    "isFork": false,
    "original": {
      "name": "",
      "githubId": 0,
      "url": ""
    },
    "isPullRequest": false,
    "pullRequestId": 0
  },
  "builds": {
    "buggyBuild": {
      "id": 421768721,
      "url": "http://travis-ci.org/fabriciofx/cactoos-jdbc/builds/421768721",
      "date": "Aug 28, 2018 5:17:20 PM"
    }
  },
  "reproductionBuggyBuild": {
    "reproductionDateBeginning": "Aug 28, 2018 5:17:42 PM",
    "reproductionDateEnd": "Aug 28, 2018 5:24:46 PM",
    "totalDuration": 418,
    "processDurations": {
      "cloning": {
        "nbSteps": 1,
        "totalDuration": 3,
        "stepNames": [
          "CloneRepository"
        ],
        "stepDurations": [
          3
        ]
      },
      "building": {
        "nbSteps": 2,
        "totalDuration": 208,
        "stepNames": [
          "CheckoutBuggyBuild",
          "BuildProject"
        ],
        "stepDurations": [
          0,
          208
        ]
      },
      "testing": {
        "nbSteps": 1,
        "totalDuration": 16,
        "stepNames": [
          "TestProject"
        ],
        "stepDurations": [
          16
        ]
      }
    },
    "projectRootPomPath": "./workspace/fabriciofx/cactoos-jdbc/421768721/pom.xml"
  },
  "patchDiff": {
    "files": {
      "numberAdded": 0,
      "numberChanged": 0,
      "numberDeleted": 0
    },
    "lines": {
      "numberAdded": 0,
      "numberDeleted": 0
    }
  },
  "commits": {
    "buggyBuild": {
      "repoName": "fabriciofx/cactoos-jdbc",
      "branchName": "mysql",
      "sha": "4beac6497236dd370803b4a6c317d8545532baeb",
      "url": "http://github.com/fabriciofx/cactoos-jdbc/commit/4beac6497236dd370803b4a6c317d8545532baeb",
      "date": "Aug 28, 2018 5:10:52 PM"
    }
  },
  "projectMetrics": {
    "numberModules": 1,
    "numberSourceFiles": 71,
    "numberTestFiles": 35,
    "numberLibrariesFailingModule": 28
  },
  "tests": {
    "failingModule": "/root/workspace/fabriciofx/cactoos-jdbc/421768721",
    "overallMetrics": {
      "numberRunning": 22,
      "numberPassing": 9,
      "numberFailing": 0,
      "numberErroring": 13,
      "numberSkipping": 0,
      "failures": [
        {
          "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
          "isError": true,
          "occurrences": 8
        },
        {
          "failureName": "org.postgresql.util.PSQLException",
          "isError": true,
          "occurrences": 5
        }
      ]
    },
    "failingClasses": [
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.script.SqlScriptTest",
        "numberRunning": 2,
        "numberPassing": 1,
        "numberFailing": 0,
        "numberErroring": 1,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.SelectTest",
        "numberRunning": 2,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 2,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.BatchTest",
        "numberRunning": 1,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 1,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.InsertTest",
        "numberRunning": 4,
        "numberPassing": 1,
        "numberFailing": 0,
        "numberErroring": 3,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.agenda.AgendaTest",
        "numberRunning": 3,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 3,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.server.PsqlServerTest",
        "numberRunning": 1,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 1,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.UpdateTest",
        "numberRunning": 1,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 1,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.server.MysqlServerTest",
        "numberRunning": 1,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 1,
        "numberSkipping": 0
      }
    ],
    "failureDetails": [
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.server.MysqlServerTest",
        "testMethod": "startAndStop",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.UpdateTest",
        "testMethod": "createTable",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.SelectTest",
        "testMethod": "any",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.server.PsqlServerTest",
        "testMethod": "startAndStop",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.agenda.AgendaTest",
        "testMethod": "addContact",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.BatchTest",
        "testMethod": "batch",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.agenda.AgendaTest",
        "testMethod": "renameContact",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.InsertTest",
        "testMethod": "insertWithKeysMysql",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.SelectTest",
        "testMethod": "select",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.agenda.AgendaTest",
        "testMethod": "findContact",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.InsertTest",
        "testMethod": "insertWithKeysPsql",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.InsertTest",
        "testMethod": "insert",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.script.SqlScriptTest",
        "testMethod": "mysql",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      }
    ]
  },
  "type": "only_fail"
}